tools/c7n_oci/c7n_oci/constants.py

```diff
@@ -12,3 +12,4 @@
 COMPARTMENT_IDS = "OCI_COMPARTMENTS"
 DEFAULT_PROFILE = "DEFAULT"
 STORAGE_NAMESPACE = "namespace_name"
+MONITORING_QUERY_RESOURCE_SIZE = 10
```

Comment:
if a constant listed here only has a single caller, please just inline it to the usage versus the indirection to a separate module.

Reply:
done